teachertool/src/transforms/runEvaluateAsync.ts

```diff
@@ -6,6 +6,11 @@ import { runSingleEvaluateAsync } from "./runSingleEvaluateAsync";
 
 export async function runEvaluateAsync(fromUserInteraction: boolean) {
     const { state: teacherTool } = stateAndDispatch();
+    const { projectMetadata } = teacherTool;
+
+    if (!projectMetadata) {
```

Comment:
Do we also need to make a check that there are criteria created? If a user loads a project without creating any criteria, will we error?

Reply:
I don't see why that would be an error, but it we could interpret it as an indication that the user is exploring the UI to learn how it works. It might make sense to view this as an opportunity to suggest adding criteria.

Reply:
Oh, it works fine without any criteria defined. We don't throw any errors.